How to handle Pyhton exceptions
